bootstrap.sh

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env bash
 
-echo "installing python"
+
+echo "installing python and postgres"
 apt update
-apt install -y python3.7 python3-pip
+apt install -y python3.7 python3-pip postgresql postgresql-contrib
+
 
 echo "installing flask"
 pip3 install --upgrade pip
-pip3 install flask
+pip3 install -r /vagrant/requirements.txt
+
+
+echo "initializing new DB"
+sudo -u postgres createdb tutor
+sudo -u postgres psql -c "ALTER ROLE postgres WITH PASSWORD 'tutor';"
```

Comment:
Consider setting the username and password into environment variables and using the same variables from the App code later as well. That way you can avoid having issues because someone mistypes the username or password somewhere.

You don't have to do this in this PR though, just open a ticket for later.

Reply:
I created a ticket about it